Cargo.toml

```diff
@@ -8,63 +8,62 @@ members = [
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 features = ["experimental-sliding-sync"]
 
 [workspace.dependencies.matrix-sdk-base]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 
 [workspace.dependencies.matrix-sdk-sqlite]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 features = ["crypto-store", "state-store"]
 
 [workspace.dependencies.matrix-sdk-store-encryption]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 
 [workspace.dependencies.matrix-sdk-test]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 
 [workspace.dependencies.matrix-sdk-ui]
-git = "https://github.com/matrix-org/matrix-rust-sdk"
-rev = "1380af4f005ee5f34607db5cbb6705f93048b85a"
+git = "https://github.com/acterglobal/matrix-rust-sdk"
+rev = "5e7a05c26f7b7029a3dc4176cf758c3cc459b04f"
 default-features = false
 features = ["e2e-encryption"]
 
 [workspace.dependencies.ruma]
-#version = "0.10.1"
-git = "https://github.com/Hywan/ruma"
-branch = "feat-sliding-sync-list-include-heroes"
+git = "https://github.com/acterglobal/ruma"
+rev = "c74a4dcc4ff6406288bbf83fa7475dc732fe3a5e"
 features = ["client-api-c", "compat-upload-signatures", "compat-user-id", "compat-arbitrary-length-ids", "unstable-msc3401"]
 
 [workspace.dependencies.ruma-client-api]
-git = "https://github.com/Hywan/ruma"
-branch = "feat-sliding-sync-list-include-heroes"
-# version = "0.18.0"
+git = "https://github.com/acterglobal/ruma"
+rev = "c74a4dcc4ff6406288bbf83fa7475dc732fe3a5e"
 
 [workspace.dependencies.ruma-common]
-git = "https://github.com/Hywan/ruma"
-branch = "feat-sliding-sync-list-include-heroes"
-#version = "0.13.0"
+git = "https://github.com/acterglobal/ruma"
+rev = "c74a4dcc4ff6406288bbf83fa7475dc732fe3a5e"
 
 [workspace.dependencies.ruma-events]
-git = "https://github.com/Hywan/ruma"
-branch = "feat-sliding-sync-list-include-heroes"
-#version = "0.28.1"
+git = "https://github.com/acterglobal/ruma"
+rev = "c74a4dcc4ff6406288bbf83fa7475dc732fe3a5e"
 
 [workspace.dependencies.ruma-macros]
-git = "https://github.com/Hywan/ruma"
-branch = "feat-sliding-sync-list-include-heroes"
-#version = "0.13.0"
+git = "https://github.com/acterglobal/ruma"
+rev = "c74a4dcc4ff6406288bbf83fa7475dc732fe3a5e"
```

Comment:
what ruma changes are needed to have our own fork?

Reply:
In fact, I needed `EmailIdentity::new()` and implemented it [here](https://github.com/ruma/ruma/commit/a02316c36a2eac042cf67ea9cad9db7bf27ac380).
But it is not useless now.
Will restore something like main branch.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1783/commits/3935bc962c9995df31b4549587d2d8d0b0fd6a1d